bare-metal/dedibox/how-to/link-dedibox-account.mdx

```diff
@@ -32,30 +32,32 @@ You can now manage your current and future Dedibox servers from the Scaleway con
-    </Message>
-5. Select the [Organization](/identity-and-access-management/iam/concepts/#organization) of your Scaleway account to which you want to link your Dedibox servers to, then click **Select Organization**.
-    <Message type="tip"> 
-      If required, click **Create a new Organization** to create a new Organization for your servers.
-    </Message>
-6. Select the personal and billing information you wish to use for your Scaleway account. You can either use the personal and billing information of your selected Scaleway Organization or the information linked to your Dedibox account. Then tick the checkboxes to confirm that:
-    * You have taken note of your login for the Scaleway console.
-    * Your Scaleway Dedibox and Scaleway Cloud account information will be updated with the information given on this page.
-    * You have understood that linked accounts can not be unlinked.
-    * Payment information will be updated and unified following these rules:
-      * If you have SEPA as your preferred payment method in either your Dedibox or Scaleway Cloud account, this SEPA then becomes the default payment method for both your Scaleway Cloud and Scaleway Dedibox invoices.
-      * If you do not have a SEPA as your preferred payment method in either account, the preferred payment method of credit card remains the same for both Scaleway Cloud and Scaleway Dedibox invoices.
-      * If you use different credit cards for Dedibox and Scaleway Cloud, the card registered with Scaleway Cloud becomes the default payment method for both Scaleway Cloud and Scaleway Dedibox invoices.
+4. We run a check to locate your existing Scaleway Account.
+   - If a Scaleway account with the same email as your Dedibox account is found, log in with your Scaleway password or request a magic link.
+   - If no related account is found, enter your account details.
         <Message type="tip">
-          Once account linking has been completed, you can modify your payment method in the [billing section](https://console.scaleway.com/billing/payment) of the Scaleway console. All existing default payment methods will be transferred to your linked account.
+          To link your Dedibox account to a different Scaleway account, click **Choose another account** and enter the new account credentials.
         </Message>
-    <Message type="note">
-      Click **Choose another account** if you want to link your Dedibox account with a Scaleway account with a different email address.
-    </Message>
-7. Click **Link this account** to complete the linking process. Your console dashboard displays and you can now manage your Dedibox servers from the Scaleway console.
+5. Choose the [Organization](/identity-and-access-management/iam/concepts/#organization) in your Scaleway account to link your Dedibox servers, then click **Select Organization**.
+      <Message type="tip">If needed, click **Create a new Organization** to create a new organization for your servers.</Message>
```

Comment:
Should it have spaces between the <> ?

Reply:
Oh indeed - it will display weird if not